### ISSUE TITLE
int64 truncated with pretty JSON

### ISSUE THREAD

**Issue by forward32** [birdayz/kaf#90]
Hi there!

Looks like `kaf` (or underlying package) truncates int64 (as JS) without `--raw` flag.

Steps to reproduce:

1. run kafka with `test` topic:
```sh
kaf topics -b localhost:9094
NAME   PARTITIONS   REPLICAS   
test   1            1
```

2. produce test data:
```sh
cat test.json | kaf produce test -b localhost:9094
```
where `test.json` looks like `{"click_id": 1669859068457630240}`

3. consume test data:
```sh
kaf consume test -b localhost:9094
Partition:   0
Offset:      0
Timestamp:   2020-01-13 20:18:17.804 +0700 +07
{
  "click_id": 1669859068457630200
}

kaf consume test --raw -b localhost:9094
{"click_id": 1669859068457630240}
```

**Comment by forward32** [comment]
I investigated this issue and found problem - it's `go-prettyjson` package.

You have to update it to `github.com/hokaccha/go-prettyjson v0.0.0-20190818114111-108c894c2c0e` to solve truncating.

**Comment by birdayz** [comment]
thanks , and thanks for investigating! i updated the lib and released v0.1.26.
